Dockerfile-dev

```diff
@@ -33,5 +33,5 @@ RUN mkdir -p /usr/src/app
 COPY . /usr/src/app/
 WORKDIR /usr/src/app
 RUN pip install --no-cache-dir --process-dependency-links -e .[dev]
-RUN bigchaindb -y configure "$backend"
+RUN bigchaindb -y configure
```

Comment:
I generally follow the advice, "If it ain't broke, don't fix it." so I wouldn't have made this change or the corresponding one in `docker-compose.yml`. We might add support for other backends in the future.

Reply:
This does, indeed, break the test. Removing this one change causes the execution to fail while building the Docker images. I'm not totally clear on why this happens, but unless removing the `"$backend"` breaks something else, I think we should pull it.

Reply:
Oh, interesting! I guess you should keep this change then.